packages/itwin/tree-widget/src/components/trees/stateless/models-tree/ModelsTree.tsx

```diff
@@ -158,11 +154,12 @@ function useSubjectModelIdsCache(imodel: IModelConnection) {
 
   const getSubjectModelIdsCache = useCallback((imodelAccess: LimitingECSqlQueryExecutor) => {
     if (prevImodelAccessRef.current !== imodelAccess) {
+      cacheRef.current?.dispose();
```

Comment:
Removed dispose method. Also removed `CacheLike` interface, since there's no use case for resetting preexisting cache at the moment - we always recreate it.